common/lib/surveygizmo_client/client.py

```diff
@@ -1,31 +1,44 @@
+"""
+Client to communicate with Survey Gizmo
+"""
 from django.conf import settings
 from surveygizmo import SurveyGizmo
 
 
 class SurveyGizmoClient(SurveyGizmo):
+    """
+    Client for Survey Gizmo
+    """
+
     def __init__(self):
-        """Instantiates the SurveyGizmo API Client.
         """
-
+        Instantiates the SurveyGizmo API Client.
+        """
         super(SurveyGizmoClient, self).__init__(
             api_version='v4',
             api_token=settings.SURVEY_GIZMO_TOKEN,
             api_token_secret=settings.SURVEY_GIZMO_TOKEN_SECRET,
         )
 
     def get_surveys(self):
-        return self.api.survey.list()
+        return self.api.survey.list()  # pylint: disable=no-member
 
-    def get_survey_responses(self, survey_id, survey_filters=[], page_no=1, results_per_page=500):
-        surveys = self.api.surveyresponse.resultsperpage(results_per_page).page(page_no)
+    def get_survey_responses(self, survey_id, survey_filters=None, page_no=1, results_per_page=500):
+        """
+        Filters survey based on the provided survey filters.
```

Comment:
We should distinguish it from `get_filtered_survey_responses` which actually it is.